src/sentry/api/endpoints/organization_member/requests/invite/index.py

```diff
@@ -79,10 +82,12 @@ def post(self, request: Request, organization) -> Response:
                 organization=organization,
                 email=result["email"],
                 role=result["role"],
-                inviter=request.user,
+                inviter_id=request.user.id,
```

Comment:
I would "lift and shift" the organization member creation and the outbox creation _into_ the `organizationmember_mapping_service`. The whole point of these services is to encapsulate the state synchronization requirements of hybrid cloud, so it makes sense to move all this logic into the service layer